dataactbroker/app.py

```diff
@@ -63,6 +63,8 @@ def teardown_appcontext(exception):
 
     @flask_app.before_request
     def before_request():
+        # Set global value for local
+        g.is_local = local
```

Comment:
if this is global now, can we stop passing it around or setting it in an init and just use the global value?